src/parameterizations/vertical/MOM_CVMix_KPP.F90

```diff
@@ -415,10 +418,17 @@ logical function KPP_init(paramFile, G, GV, US, diag, Time, CS, passive, Waves)
                    '\t RW16     = Function of Langmuir number based on RW16',    &
                    default='CONSTANT')
     select case ( trim(string))
-      case ("CONSTANT") ; CS%LT_K_METHOD = LT_K_MODE_CONSTANT
-      case ("VR12")     ; CS%LT_K_METHOD = LT_K_MODE_VR12
-      case ("RW16")     ; CS%LT_K_METHOD = LT_K_MODE_RW16
-      case default      ; call MOM_error(FATAL,"KPP_init: "//&
+      case ("CONSTANT")
+        CS%LT_K_METHOD = LT_K_MODE_CONSTANT
+        langmuir_mixing_opt = 'LWF16'
+      case ("VR12")
```

Comment:
Is there any documentation for these options, e.g., `LWF16`? If so, please add a comment. Thanks!

Reply:
Thanks, @sanAkel. @alperaltuntas has addressed your suggestions in https://github.com/NOAA-GFDL/MOM6/pull/1491/commits/11c550f55a2da9b60577d40dc6f7504e580a71aa